src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php

```diff
@@ -27,7 +29,7 @@ class Query extends AbstractQuery
     /** @var Connection */
     protected $connection;
 
-    /** @var resource[] */
+    /** @var (resource|Result)[] */
```

Comment:
This PHPdoc is slightly inaccurate as we should be able to expect all values to be a resource or the Result object, depending on PHP being >= 8.1 or not. Same belongs to the getResources function.

Reply:
fixed. Thanks